pages/early-access/_hero.html

```diff
@@ -2,15 +2,14 @@
   <div class="row">
     <div class="col-xs-12 text-center">
       <h1>
-        <label style="display: inline-flex; align-items: center; margin-bottom: 0">{{ page.title }}
+        {{ page.title }}
+        <label style="display: inline-flex; align-items: center; position: relative; top: -10px;">
```

Comment:
Are these changes to the Patcher early access pages intentional?

Reply:
Hi @brikis98 , yes - I noticed some CSS issues there, so I updated it.

Reply:
Roger, just checking!